ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/Cli.scala

```diff
@@ -226,18 +209,41 @@ object Cli {
         config.fold(error => throw new IllegalArgumentException(error), identity)
       }
 
-    private def listOfTemplateIds(listOfIds: String): Either[String, List[Identifier]] =
+    private def filters(listOfIds: String): Either[String, Map[String, Option[List[Identifier]]]] =
       listOfIds
-        .split('|')
+        .split('&')
```

Comment:
Can you also use a comma?

`filters=party1,template1,template2+party2`

Reply:
@rautenrieth-da In the current state of CLI parser I cannot use commas, because commas are used for separating different stream parameters, e.g.:
```
--consume-stream=stream-type=transactions,name=stream-1,filters=00001@template1+00002@template3@template4,max-delay=1
```

As a side note I'd like to mention that I'm currently working on making it possible to run both submission and stream reading phases in a single session. Such workflow is going to be described in a yaml file which might be more convenient for using filters. For this reason I would say that describing a stream with CLI parameters will be just an addition for scripting purposes because using a yaml file will be cleaner.